BUG-BOUNTY-DETAILS.md

```diff
@@ -2,13 +2,13 @@
 
 Thank you for visiting the WeiFund Bug Bounty.
 
-WeiFund is crowdfunding infrastructure on and for the Ethereum ecosystem. Successful bug hunters will be rewarded with both ether and recognition on our website and github page (See Rewards & Rules section for details).
+WeiFund is crowdfunding infrastructure on and for the Ethereum ecosystem. Successful bug hunters will be rewarded with up to $5,000 in ether and recognition on our website and github page (See Rewards & Rules section below for details).
```

Comment:
As a bounty hunter, I really just want to know how much I could earn right away.